spaces/domains.json

```
@@ -498,5 +498,8 @@
   "signal.paladin.vote": "palvote.eth",
   "dao.cronosfrens.io": "cronosfrens.io",
   "vote.tresleches.finance": "3leches.eth",
+  "gov.fxcm.xyz": "symbolspace.eth",
   "gov.wagumi.xyz": "wagumi.eth"
+  
+    
```

Please remove extra lines.
```suggestion
  "gov.fxcm.xyz": "symbolspace.eth",
  "gov.wagumi.xyz": "wagumi.eth"
```